lib/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace sleep {
+    // allows for import * as sleep from "sleep-promise";
```

Comment:
@kachkaev would a "module" field in package.json with a reference to a file that has a default export as an esm help typescript?

I would prefer it being imported as `import sleep from 'sleep-promise';` instead of `import * as sleep from 'sleep-promise';`.